website/docs/developer/research/research-reconciliation.mdx

```diff
@@ -115,31 +115,40 @@ seemed a bit challenging and did not appear to provide a real benefit.
+| collection_id       | `string`    | Collection ID from Cumulus that refrences the Collections table.            | None                                     |
+| cumulus_create_time | `timestamp` | Date and time data was originally ingested into cumulus                     | None                                     |
+| cumulus_granule_id  | `string`    | granuleId that is provided by Cumulus.                                      | Must be unique per collectionId          |
+| execution_id        | `string`    | Step function execution ID from AWS.                                        | Unique ID automatically generated by AWS |
+| ingest_time         | `timestamp` | Date and time in UTC that the data was originally ingested into ORCA        | None                                     |
+| last_update         | `timestamp` | Date and time in UTC that information was updated.                          | None                                     |
 
 Note that the combination of the Cumulus Granule ID (cumulus_granule_id) and
 the Cumulus Collection ID (collection_id) is unique in the
 [Cumulus granules table](https://github.com/nasa/cumulus/blob/master/lambdas/db-migration/src/migrations/20201014105058_create_granules_table.ts).
 
 #### File Metadata Object
 
-| Attribute Name           | Data Type | Description                                       | Notable Items   |
-| ------------------------ | --------- | ------------------------------------------------- | --------------- |
-| name                     | `string`  | Name of the file. (MOD14A1.061.2020245.hdf)       | Must be unique per granuleId |
-| orca_archive_location    | `string`  | ORCA S3 Glacier bucket the file resides in.       | |
-| cumulus_archive_location | `string`  | Cumulus S3 bucket the file is expected to reside in. | |
-| key_path                 | `string`  | Full S3 path to the file including the file name. | Must be unique per archive location |
+| Attribute Name           | Data Type | Description                                                               | Notable Items                                        |
+| ------------------------ | --------- | ------------------------------------------------------------------------- |--------------------------------------- ------------  |
+| id                       | `string`  | Internal ORCA file ID                                                      | None                                                |
+| granule_id               | `string`  | Granule that the file belongs to refrences the internal ORCA granule ID.   | None                                                |
+| name                     | `string`  | Name of the file. (MOD14A1.061.2020245.hdf)                                | Must be unique per granuleId                        |
+| orca_archive_location    | `string`  | ORCA S3 Glacier bucket the file resides in.                                | None                                                |
```

Comment:
The example suggests that orca_archive_location and cumulus_archive_location are actually complete urls, not just the bucket name.

Reply:
"The url of the file within an ORCA S3 Glacier bucket."?